engine/worker/run_plugin.go

```diff
@@ -123,47 +128,45 @@ func startGRPCPlugin(ctx context.Context, pluginName string, w *currentWorker, p
 	cmd := binary.Cmd
 	if _, err := exec.LookPath(cmd); err != nil {
 		cmd = path.Join(w.basedir, cmd)
-		_, err = exec.LookPath(cmd)
-		if err != nil {
-			return nil, sdk.WrapError(err, "plugin:%s unable to start GRPC plugin, binary command not found.", pluginName)
```

Comment:
Why removing all the Wraps ? If the goal is to remove the printed call stack from sdk errors, you could use the errors.Wrap or Wrapf func that are nearly the same as fmt.Errorf("more data about the error: %v", err).

Reply:
CheckErr with wrap didn't work on my local tests. I put Errorf everywhere in this func to fix other bug with logs, I'll try to reput WrapError (probably not on this branch if not done today, but in a second commit, I have to understand why Wrap are not ok)

Reply:
@richardlt ok, I understand.

this is wrong:

```go
if errBi != nil || binary == nil {
			return nil, sdk.WrapError(errBi, "plugin:%s Unable to get plugin binary infos... Aborting", pluginName)
		}
```

if binary is nil, then wrapError wraps a nil err, then the caller don't see the err.... I replace only this wrap.

Reply:
@richardlt fixed